docs/docs/services/qix-engine/access-control.md

```diff
@@ -132,12 +246,10 @@ However, it is recommended that you are consistent in the order in which you def
 resources and conditions as this simplifies troubleshooting. When using multiple conditions,
 you can group two conditions by wrapping them in parentheses.
 
-**Example**
+##### Example
```

Comment:
It's called `Examples` (plural) elsewhere

Reply:
Can we add more examples on `Conditions`? I found this section quite hard to get a grip of.